school-sevice/src/test/java/liga/school/sevice/service/SchoolServiceImplIntegrationTest.java

```diff
@@ -0,0 +1,390 @@
+package liga.school.sevice.service;
+
+import liga.school.sevice.domain.entity.School;
+import liga.school.sevice.domain.repository.SchoolRepository;
+import liga.school.sevice.exception.SchoolNotFoundException;
+import liga.school.sevice.exception.StudentNotFoundException;
+import liga.school.sevice.transport.dto.SchoolCreateDto;
+import liga.school.sevice.transport.dto.SchoolFindDto;
+import liga.school.sevice.transport.dto.SchoolOutComeDto;
+import liga.school.sevice.transport.dto.SchoolUpdateDto;
+import org.assertj.core.util.Sets;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.persistence.EntityNotFoundException;
+import java.util.*;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toSet;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.anySet;
+import static org.mockito.Mockito.when;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@Transactional
+public class SchoolServiceImplIntegrationTest {
+
+    @Autowired
+    private SchoolService schoolService;
+
+    @MockBean
+    private StudentService feignService;
+
+    @Autowired
+    SchoolRepository schoolRepository;
+
+    @Test
+    @Sql("/scripts/initSchools.sql")
+    public void testGetAll() {
+        PageRequest pageable = PageRequest.of(0, 10, Sort.Direction.ASC, "id");
+        SchoolFindDto schoolFindDto = SchoolFindDto
+                .builder().build();
+
+        Page<SchoolOutComeDto> result = schoolService.getAll(schoolFindDto, pageable);
+
+        assertEquals(5, result.getContent().size());
```

Comment:
also add checking of content and DB, read from db schools and compare with response